strings/en_US/messages.po

```diff
@@ -4067,3 +4073,164 @@ msgstr "Please tell us about any special requirements that we might need to know
 
 msgid "Not attending"
 msgstr "Not attending"
+
+msgid "Your tickets are now awaiting approval"
+msgstr "Your tickets are now awaiting approval"
+
+msgid "You will be notified about future events hosted by this dojo"
+msgstr "You will be notified about future events hosted by this dojo"
+
+msgid "You will be notified when the organizer approves your request."
+msgstr "You will be notified when the organizer approves your request."
+
+msgid "Booking Request Sent"
+msgstr "Booking Request Sent"
+
+msgid "Fully booked"
+msgstr "Fully booked"
+
+msgid "Confirm booking for {totalBooked} ticket(s)"
```

Comment:
Would be nice to use plural for this, but iirc we don't have plural support in frontend?

Reply:
confirming, no we don't :(